concepts/Common/Security Considerations/20 HTML Encoding/20 Encoding in Templates.md

```diff
@@ -23,11 +23,10 @@ Angular, Vue, and React always encode values interpolated in templates. Without
         content: "Tab content" 
     }];
 
-<a href="https://codepen.io/romantsukanov/pen/vYKjzQw?editors=1010" target="_blank">View on CodePen</a>
 
 When you insert unencoded content, it can open your application to XSS attacks:
 
-![DevExtreme Tabs with disabled HTML encoding](/images/UiWidgets/tabs-itemTemplate-unsafe-html.png)
+![DevExtreme Tabs with disabled HTML encoding](/images/UiWidgets/tabs-itemTemplate-unsafe-html-wo-codepen.png)
 
 The encoded content is interpreted and displayed as text:
 
```

Comment:
Let's rename the articles to avoid -ing:

Encoding User Input -> Encode User Input
Encoding in Templates -> Encode Template Data